README.md

```diff
@@ -1,143 +1,110 @@
-# Conduit PostgreSQL Connectors 
+# Conduit Connector PostgreSQL
```

Comment:
Nitpick: I just noticed this, and to me it's more natural to say: a Pg connector, a Kafka connector etc. Or: Conduit connector for Pg.

Reply:
Yeah I noticed this as well, I used this just because we use the same format in other connectors as well and to keep it consistent. I'm all for changing this title, but then we should do it across the board. I'd say we can do this after the release since it's only the readme.